### PR TITLE
weekly highlights 2022-06-03

### PR DESCRIPTION
**Pull request by gianfranco-l**
weekly highlights 2022-06-03

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/changes/highlights/2022-06-03.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2022-06-03T10:00:00
+title: Highlights for the week ending June 03, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
```
Comment (weatherhog) [inline src/content/changes/highlights/2022-06-03.md:9]:
```suggestion
## General

With [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) release [v2.11.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#2110---2022-04-22) we added fixes for [CVE-2021-25745](https://github.com/advisories/GHSA-pvmg-xgmx-9mxh) and [CVE-2021-25746](https://github.com/advisories/GHSA-79xv-4hmm-pw72). Please also see the upstream [issue](https://github.com/kubernetes/ingress-nginx/issues/8502) for `CVE-2021-25745` and the upstream [issue](https://github.com/kubernetes/ingress-nginx/issues/8503) for `CVE-2021-25746`. It is highly recommend to update all [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) installations.
```

File: src/content/changes/highlights/2022-06-03.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+## User interfaces
+
+
+## Documentation
+
+
+## Apps
+
```
Comment (ubergesundheit) [inline src/content/changes/highlights/2022-06-03.md:17]:
```suggestion
- [kong-app](https://github.com/giantswarm/kong-app) version [v2.10.0](

[linkerd2-app](https://github.com/giantswarm/linkerd2-app) version [v0.7.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#2100---2022-06-02) updates to upstream version [2.8.2](https://github.com/Kong/charts/releases/tag/kong-2.8.2) and changes default values to more evenly schedule of pods.

```

**Comment by gianfranco-l** [comment]
@marians @anvddriesch @alex-dabija @cornelius-keller anything from your side?

File: src/content/changes/highlights/2022-06-03.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+With [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) release [v2.11.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#2110---2022-04-22) we added fixes for [CVE-2021-25745](https://github.com/advisories/GHSA-pvmg-xgmx-9mxh) and [CVE-2021-25746](https://github.com/advisories/GHSA-79xv-4hmm-pw72). Please also see the upstream [issue](https://github.com/kubernetes/ingress-nginx/issues/8502) for `CVE-2021-25745` and the upstream [issue](https://github.com/kubernetes/ingress-nginx/issues/8503) for `CVE-2021-25746`. It is highly recommended to update all [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) installations.
+
+## User interfaces
```
Comment (marians) [inline src/content/changes/highlights/2022-06-03.md:14]:
```suggestion
## User interfaces

You can now inspect permissions for service accounts via the web UI, in addition to users and groups.

```

File: src/content/changes/highlights/2022-06-03.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## Apps
+- [kong-app](https://github.com/giantswarm/kong-app) version [v2.10.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#2100---2022-06-02) updates to upstream version [2.8.2](https://github.com/Kong/charts/releases/tag/kong-2.8.2) and changes default values to more evenly schedule of pods.
+
+
+## Documentation
+
+
```
Comment (marians) [inline src/content/changes/highlights/2022-06-03.md:22]:
```suggestion
## Documentation

We completely updated our article on [labelling workload clusters](https://docs.giantswarm.io/advanced/labelling-workload-clusters/) and added information on a new recommended label for indicating a cluster's importance.
```

**Review by marians — APPROVED** [review]

**Pull request merged**